cvat-ui/src/containers/actions-menu/actions-menu.tsx

```diff
@@ -95,31 +103,40 @@ function ActionsMenuContainer(props: OwnProps & StateToProps & DispatchToProps):
         openRunModelWindow,
         exportTask,
         openMoveTaskToProjectWindow,
+        showWorkoutModal,
     } = props;
 
-    function onClickMenu(params: MenuInfo): void {
-        const [action] = params.keyPath;
-        if (action === Actions.EXPORT_TASK_DATASET) {
-            showExportModal(taskInstance);
-        } else if (action === Actions.DELETE_TASK) {
-            deleteTask(taskInstance);
-        } else if (action === Actions.OPEN_BUG_TRACKER) {
-            window.open(`${taskInstance.bugTracker}`, '_blank');
-        } else if (action === Actions.RUN_AUTO_ANNOTATION) {
-            openRunModelWindow(taskInstance);
-        } else if (action === Actions.EXPORT_TASK) {
-            exportTask(taskInstance);
-        } else if (action === Actions.MOVE_TASK_TO_PROJECT) {
-            openMoveTaskToProjectWindow(taskInstance.id);
-        }
-    }
+    const onClickMenu = useCallback(
+        (params: MenuInfo) => {
+            const [action] = params.keyPath;
+            if (action === Actions.EXPORT_TASK_DATASET) {
```

Comment:
switch?

Reply:
Well it's an original syntax, so I'd like to avoid to change it to switch syntax because... It could produce too many conflicts in case of merging with new commits from `cvat` repo.